app/views/posts/index.html.erb

```diff
@@ -1,2 +1,34 @@
 
 <h1>Here is a list of posts for a given user</h1>
+
+  <div class="container">
+      <div class="row">
+          <div class="col-3 border my-3 p-3">
+              
+          </div>
+          <div class="col-9">
+              <div class ="d-flex justify-content-around align-items-baseline border m-3 p-3">
+                <h1><%= @user.name %></h1>
+                <p>Number of posts : <%= @user.posts_counter %></p>
+            </div>
+          </div>
+      </div>
+      <div class="row">
+         <div class="col-12 border ">
+              <h1>Bio</h1>
+            <p><%= @user.bio %></p>
+          </div>
+      </div>
+      <% @posts.each do |post| %>
+      <div class="row mt-3">
+         <div class="col-12 border ">
+              <h1><%= post.title %></h1>
+            <p><%= post.text %></p>
+            <div class ="d-flex flex-row-reverse">
```

Comment:
- [ ] I should be able to visit the post's show page by clicking on the post's name.

Reply:
Please check my previous comment